recipes/stx/all/conandata.yml

```
@@ -0,0 +1,10 @@
+sources:
+    "1.0.1":
+        url: https://github.com/lamarrr/STX/archive/v1.0.1.tar.gz
+        sha256: C491C888C56B9F800EB7C17A30C81AFCF2D9FEC4CDAE9A73CD1145698A645DD9
+patches:
+    "1.0.1":
+        - patch_file: patches/0001-abseil_find_package.patch
+          base_path: source_subfolder
+        - patch_file: patches/0002-cmake_module_path.patch
+          base_path: source_subfolder
```

```suggestion
  "1.0.1":
    url: https://github.com/lamarrr/STX/archive/v1.0.1.tar.gz
    sha256: C491C888C56B9F800EB7C17A30C81AFCF2D9FEC4CDAE9A73CD1145698A645DD9
patches:
  "1.0.1":
    - patch_file: patches/0001-abseil_find_package.patch
      base_path: source_subfolder
    - patch_file: patches/0002-cmake_module_path.patch
      base_path: source_subfolder
```

minor change: .editorconfig enforces 2 spaces in yml